utils/textmate/info.plist

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
```

Comment:
I had some troubles with the copyright checker with this one. It suggests I use `#` for comments, but those are not valid XML comments

Reply:
In /.pre-commit-config.yaml, should be something like (added under `check-copyright`):

```
      - --custom_format
      - '\.js$'
      - '<!--'
      - ''
      - '-->'
```